modules/tinymce/src/core/main/ts/api/html/Styles.ts

```diff
@@ -263,6 +265,16 @@ const Styles = (settings: StylesSettings = {}, schema?: Schema): Styles => {
               value = value.toLowerCase();
             }
 
+            // Convert RGB/RGBA colors to HEX
+            if (typeof settings.force_hex_color === 'string' && settings.force_hex_color !== 'off') {
```

Comment:
Should probably use the `Type` module from Katamari instead. Do we need the string check actually?
```suggestion
            if (Type.isString(settings.force_hex_color) && settings.force_hex_color !== 'off') {
```

Reply:
We do because it's optional. If it was made non-optional then the string check could be removed.

Reply:
I swapped it over to use `Type.isString`